pkg/features/antrea_features.go

```diff
@@ -158,6 +158,10 @@ const (
 	// alpha: v1.15
 	// Enable layer 7 flow export on Pods and Namespaces
 	L7FlowExporter featuregate.Feature = "L7FlowExporter"
+
+	// alpha: v1.15
```

Comment:
for now, you can have v2.0 as a placeholder (v1.15 was already released...)

Reply:
Thanks, It's been updated.